### ISSUE TITLE
New features for cmb_convergence tracer

### ISSUE THREAD

**Issue by gfabbian** [LSSTDESC/sacc#101]
When performing inference from the CMB convergence auto spectrum, one has to deal with noise biases (N0, N1 etc.). 
These carry a cosmology dependence that needs to be accounted for. A common way to do that is to account for it during inference through a Taylor expansion that requires precomputing (and storing) their derivatives w.r.t. CMB and CMB convergence power spectra. These are normally additional matrices (potentially non-squared depending on how binning is managed).

It would be useful to store these structures within a single tracer class as additional fields but this may require adding extra features/methods specific to CMB lensing alone. It might then make sense to write CMB lensing as a separate specific tracer (inheriting the general class). Would that be the best/correct way to deal with this or do developers think a better/easier solution is possible?

**Comment by joezuntz** [comment]
Yes, that seems to make sense. We probably don't want it to be mandatory, since simpler forecasting work will presumably want to omit it.

**Comment by gfabbian** [comment]
Ok sounds great. I'll fork and propose a merge request when finalized then.
